libsolidity/codegen/MultiUseYulFunctionCollector.cpp

```diff
@@ -32,11 +32,11 @@ using namespace solidity::frontend;
 string MultiUseYulFunctionCollector::requestedFunctions()
 {
 	string result;
-	for (auto const& f: m_requestedFunctions)
+	for (auto const& [name, code]: m_requestedFunctions)
 	{
-		solAssert(f.second != "<<STUB<<", "");
+		solAssert(code != "<<STUB<<", "");
```

Comment:
Does the compiler warn that `name` is unused here? I've had this question for some time myself :-)

Reply:
Good point. It does not. Which is weird because I remember having problems with unused variables in destructuring.